graphcmd/main.go

```diff
@@ -15,6 +15,7 @@ var (
 	flaggedField         = flag.String("flagged-field", "customfield_10002", "the name of the custom field for impediment flagging")
 	sprintsField         = flag.String("sprints-field", "customfield_10007", "the name of the custom field for Greenhopper sprints")
 	epicLinkField        = flag.String("epic-link-field", "customfield_10200", "the name of the custom field for Epic Link")
+	useLiveFiles         = flag.Bool("use-live-files", false, "use live templates and assets instead of embedding, for faster iteration during local development")
```

Comment:
This is useful, but it would be ideal if this function is available for development only, and not in the final packaged binary (including templates, assets, etc.) - it's not something needed by a user of jiragraph who is not also a developer.

I know it's annoying that the build process for jiragraph requires rebuilding when any frontend files have changed. I know modern frontend development stacks typically come with a web server capable of rebuilding assets on file-change, which is very convenient.

@benjamincudi I'm thinking about introducing a new "run" Make target that leverages some kind of auto-rebuilding when frontend files change, probably Node, since that is what is typically used for React apps. It should be possible to have this server rebuild assets that would otherwise be served from `dist` and reverse-proxy to to the Go server for template-driven HTML and REST resources. I think this approach would be better than having a flag - the local developer could just do `make run` to benefit from what `use-live-files` does. Do you think it's a good idea? Any advice for me?

Reply:
@andrei-m I'm may not understand the specific concern here, but the code prior to this PR already picks up changes to templates and js/css as long as you refresh your browser- no rebuild of the binary needs to happen, but the js does need to be manually rebuilt as you noted for that part to be detected

Is the concern that the built binary will still contain this flag even for a "production" build? If so, I'm not sure I have any good ideas for how to avoid that 😬 I could do some investigation to see if anyone has done some switching on build variables other than goos/goarch, but that seems risky

Let me know if I misunderstood the concern, I'll see if there's any good options for restricting this behavior

Reply:
Yes, what I had in mind instead of the flag is a separate Make target for the "use-live-files" behavior using a server that can rebuild the JS on-file-change.

Reply:
I'm not sure if Go has a good mechanism for supporting that, but it could be an environment variable instead? I mostly thought a flag was more ergonomic, but I'm coming up short on a way to actually eliminate this code path entirely in a "prod" build. An environment variable would definitely be less discoverable though

Reply:
It turns out build tags are a real thing, I'll switch to something using those 👍 i somehow thought those only were used for OS differences

Reply:
@andrei-m let me know if this works for what you had in mind - checked locally with passing `-tags dev` to both the `install` and `go build` approach for development and found it working, and verified that a build without that (e.g a "prod" build) would yell about the unknown flag
https://github.com/andrei-m/jira-graph/pull/18/commits/467dcbe66c06826db9b4d751b6f4276b9e38cf38

Reply:
This looks good; thank you!